### DISCUSSION TITLE
Test Console

### DISCUSSION THREAD

**Discussion by charlesbetros** [jonathanpeppers/dotnes#39] [Ideas]
I think a console app that runs the transpile task might be useful. I wrote a really simple one for my local dev. We could do some assembly output comparisons for basic acceptance tests. I haven't put much thought in to selecting a project to "run".

**Comment by jonathanpeppers** [comment]
If you have one, we could just name the project `dotnes.compiler` and make the resulting binary named `dotnes.exe`?

We could start where it literally just uses `args[0]` and `args[1]` with help text?

For a full-fledged CLI options, I would try either Mono.Options or System.CommandLine, but that could come later!

**Reply by charlesbetros** [reply]
PR created. Closing this discussion.
